crates/chain/tests/common/mod.rs

```diff
@@ -4,6 +4,22 @@ mod tx_template;
 #[allow(unused_imports)]
 pub use tx_template::*;
 
+#[allow(unused_macros)]
+macro_rules! anchor {
+    ($height:expr, $hash:literal) => {
+        (
+            (
+                bdk_chain::bitcoin::Txid::all_zeros(),
+                bdk_chain::BlockId {
+                    height: $height,
+                    hash: bitcoin::hashes::Hash::hash($hash.as_bytes()),
+                },
+            ),
+            bdk_chain::BlockTime::new(100),
```

Comment:
`()` might be a better default for tests. Unless we need blocktime for the tests.